### PR TITLE
fix: <svg> attribute width="<N>rem" errors in Safari

### PR DESCRIPTION
**Pull request by dkochetkov**
Сhanged the with and height value for all svgs used in the LeftMenu & Onboarding. It was changed from <N>rem to <N*10>px. Rem was leading to the error message Error: Invalid value for <svg> attribute width="<N>rem" in the safari browser.

### What does it do?

Changed the with and height value for all svgs used in the packages/core/admin/admin/src/components/LeftMenu.tsx LinkIcon component. It was changed from 2rem to 20px.
Changed the with and height value for all svgs used in the packages/core/admin/admin/src/components/Onboarding.tsx Play component. It was changed from 1.2rem to 12px.

### Why is it needed?

Rem was leading to the error message Error: Invalid value for <svg> attribute width="<N>rem" in the safari browser.

### How to test it?

Run new project in dev mode, open admin panel (Home page) in Safari browser, open console and watch errors.
Click Help button and watch errors in console.

### CODE REVIEW

**Comment by alexandrebodin** [comment]
Hi @dkochetkov thank you for your Pull request 💯 

 We will do our best to review it as soon as we can.

File: packages/core/admin/admin/src/components/LeftMenu.tsx [hunk 1]
```diff
@@ -100,7 +100,7 @@ const LeftMenu = ({ generalSectionLinks, pluginsSectionLinks }: LeftMenuProps) =
                       aria-label={labelValue}
                     >
                       <NavLink.Icon label={labelValue}>
-                        <LinkIcon width="2rem" height="2rem" fill="neutral500" />
+                        <LinkIcon width="20px" height="20px" fill="neutral500" />
```
Comment (simotae14) [inline packages/core/admin/admin/src/components/LeftMenu.tsx:103]: I would prefer
```suggestion
                        <LinkIcon width="20" height="20" fill="neutral500" />
```

**Review by simotae14 — CHANGES_REQUESTED** [review]
Hey @dkochetkov 
thanks a lot for your contribution and the great work done. 💪🏽 

Can I ask you to modify your solution with my suggestions and also can you create a different branch instead of develop for your solution, something like `fix/svg-error-in-safari`
Thanks a lot

**Comment by dkochetkov** [comment]
Closed by #21550

**Pull request closed without merging**